src/proto/http-relay/http_over_rpc.proto

```diff
@@ -46,4 +46,5 @@ message HttpResponse {
   optional bytes body = 3;
   optional bool eof = 5;
   repeated HttpHeader trailer = 6;
+  optional int64 backend_duration_ms=7;
```

Comment:
should this be a google/protobuf/duration.proto ?
https://pkg.go.dev/google.golang.org/protobuf/types/known/durationpb

Reply:
technically yes, but I don't want to add the extra dependency here. Do you think it is worth it?